### PR TITLE
test commit

### PR DESCRIPTION
**Pull request by jquintanan**
testing branches and commuits

### CODE REVIEW

File: SortingAlgorithms/index.html [hunk 1]
```diff
@@ -10,11 +10,11 @@
 
 	<table border="1">
 		<tr>
-			<td>1</td>
-			<td>2</td>
-			<td>3</td>
-			<td>4</td>
-			<td>5</td>
+			<td>1 Hi</td>
```
Comment (jquintanan) [inline SortingAlgorithms/index.html:13]: All of these were needed because the table seemed TOO boring!

**Review by jquintanan — COMMENTED** [review]
Check the new more exciting table!

File: SortingAlgorithms/index.html (hunk 1 above)
Comment (DeanDean12) [reply SortingAlgorithms/index.html:13]: Looks good!

**Review by DeanDean12 — APPROVED** [review]
Approved!

**Pull request merged**